lib/mongoid/interceptable.rb

```diff
@@ -43,6 +43,8 @@ module Interceptable
       # @api private
       define_model_callbacks :persist_parent
 
+      define_model_callbacks :commit, :rollback, only: :after
```

Comment:
Note Rails also has [these methods](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html#method-i-after_save_commit):


```
after_create_commit
after_update_commit
after_save_commit
after_destroy_commit
```

Reply:
@johnnyshields Good catch, thank you! I think we forgot about them when planing this featue. I create a ticket to implement them in score of this epic - https://jira.mongodb.org/browse/MONGOID-5540